app/backend/src/clj/teet/land/owner_opinion_export.clj

```diff
@@ -14,14 +17,133 @@
            [(missing? $ ?opinion :meta/deleted?)]
            :in $ ?activity ?type]
          db activity type)
-    (db-api-large-text/with-large-text #{:cooperation.response/content :cooperation.opinion/comment})
+    (db-api-large-text/with-large-text #{:land-owner-opinion/authority-position :land-owner-opinion/body})
     (map first)))
 
-(defn summary-table [db activity type]
-  (let [opinions (opinions-by-type db activity type)]
+(def tr* #(tr [:land-owner-opinion :export %]))
+
+(defn jrk-nr
```

Comment:
As Erno said, functions in this ns could use descriptive comments for future generations.